src/VisualStudio/Core/Def/Microsoft.VisualStudio.LanguageServices.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    $CommonMetadataElements$
+    <dependencies>
+      <group targetFramework="net472">
+        <dependency id="Microsoft.CodeAnalysis.Common" version="$Version$" exclude="Build,Analyzers" />
```

Comment:
Does `"Build,Analyzers"` mean that this dependency will exist only as a runtime component?

Reply:
https://docs.microsoft.com/en-us/nuget/reference/nuspec#dependencies-element

I copied the list from the previous version of the package. Turns out this is the default and can be removed. `build` means build targets/props files.